api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user) who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.

```

Comment:
I actually think this intro is OK ( but also not sure about the last sentence).  Maybe to assuage Jimaco, we could say that "Groups are collections of different principals, primarily users (link)."  This makes it broader (and more accurate) for all of our group types, although I worry that "principals" makes it a bit techie.  Then we can qualify it further in the next sentence in terms of how groups are used - i.e. to govern access to resources in Microsoft services or in your apps.  How does that sound?